lib/literature/templates/layout/root.html.heex

```diff
@@ -31,6 +31,7 @@
     <%= render_publication_language_tags(@publication, current_url(@conn, %{})) %>
     <%= render_post_language_tags(assigns[:post], @publication) %>
     <!-- END Language tag -->
+    <%= render_profile_page_schema(assigns) %>
```

Comment:
@jdamanalo not sure this should be in the root, as this tag should be shown on the profile page this might need to move into martide's template files